### PR TITLE
multipart upload through CLI

### PR DESCRIPTION
**Pull request by cdebourcy**
goes with https://github.com/chanzuckerberg/idseq-cli/pull/18

### CODE REVIEW

**Comment by jshoe** [comment]
looks fine after you fix that file.parts reference?

**Comment by cdebourcy** [comment]
Good catch! Haven’t added the schema migration

File: app/models/sample.rb [hunk 1]
```diff
@@ -83,7 +84,12 @@ def set_presigned_url_for_local_upload
     input_files.each do |f|
       if f.source_type == 'local'
         # TODO: investigate the content-md5 stuff https://github.com/aws/aws-sdk-js/issues/151 https://gist.github.com/algorist/385616
-        f.update(presigned_url: S3_PRESIGNER.presigned_url(:put_object, bucket: SAMPLES_BUCKET_NAME, key: f.file_path))
+        parts = f.parts.split(", ")
```
Comment (yunfangjuan) [inline app/models/sample.rb]: split by comma and space?

File: app/models/sample.rb [hunk 2]
```diff
@@ -267,6 +273,27 @@ def check_host_genome
     s3_preload_result_path.strip!
   end
 
+  def concatenate_input_parts
```
Comment (yunfangjuan) [inline app/models/sample.rb]: you are doing this for every sample.save?

Comment (cdebourcy) [reply app/models/sample.rb]: thanks, added a condition

File: app/models/sample.rb (hunk 1 above)
Comment (cdebourcy) [reply app/models/sample.rb]: yes, is that ok?

**Pull request merged**